docs/framework/get-started/system-requirements.md

```diff
@@ -26,6 +22,8 @@ manager: "wpickett"
 
 The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop apps for the .NET Framework have a separate set of requirements.
 
+All .NET Framework versions since .NET Framework 4 are in-place updates. In other words, if there's a later version installed on the machine already, you don't have to install the previous versions. Similarly, if you install a later version, you don't have to first uninstall the previous versions. And, if an OS already comes pre-installed with a particular .NET Framework version, you cannot install previous 4.x versions on the same machine.
```

Comment:
This is worth modifying somewhat and doing as a note; I don't remember, though, if notes support bulleted lists, and if Important is a note category. The text that I'd change it do is:
All .NET Framework versions since .NET Framework 4 are in-place updates; only a single 4.x version can be present on a system. In addition, particular versions of the .NET Framework are pre-installed on some versions of the Windows operating system. This means that:
- If there's a later version installed on the machine already, you cannot install a previous version.
- If the OS comes pre-installed with a particular .NET version, you cannot install a previous 4.x version on the same machine.
- If you install a later version, you don't have to first uninstall the previous version.

It probably would be good to make the same change to the Install the .NET Framework for developers topic; too many customers seem to overlook the comment. I can open a separate PR for that tomorrow (and also incorporate the other changes in the version table).